RecoHI/HiTracking/python/hiDetachedTripletStep_cff.py

```diff
@@ -161,8 +160,7 @@
     numHitsForSeedCleaner = cms.int32(50),
```

Comment:
Why not removing the explicit type also here and below (and above for `maxDPhiForLooperReconstruction`, `maxPtForLooperReconstruction`)?

Reply:
@perrotta I agree, this should be fixed. I'm wondering what the best timeline is: we could test this PR as is in the IB over the weekend, to see if any issues show up in other wfs, before the next pre-release, or I could apply the changes now and that will likely delay things a bit (b/c of re-testing and re-getting signatures). If you think this could be merged already now, I would go for the first option (and will make a separate PR on Monday to fix the explicit-type changes I missed); if you prefer otherwise, I can apply the changes already to this PR.